Sophia Sun Project Pitch

```diff
@@ -0,0 +1,244 @@
+
+# What is that person's experience like now?
+
+Right now, to play the accompaniment of a song, a user has to:
+Look up the chords on a site like https://tabs.ultimate-guitar.com
+Read the letters and numbers that make up the name of a chord
+Understand what notes are in a chord, given its name 
+FIgure out how those chords can be played on the piano
+
+# How would their experience be better if you could help them?
+If I could help them, they would be able to directly translate from the name of a chord to seeing a depiction of the chord, dictated on a musical staff. 
+
+For example, my language would parse the name of the chord given by the user and translate the name of the chord (such as “GDom7”) into its notation on a musical staff (on sheet music). On the sheet music, the user would see 4 notes representing “G B D F”, which are the notes that make up the “GDom7” chord. 
+
+The users would be able to more quickly play the accompaniment part of a song they like on the piano! Instead of having to search up the chords and try to decipher how to play each chord, my DSL will help them translate the name of the chord to an image of the musical staff with the chord notated on it.  
+
+### Why a language?
+# Why is a DSL appropriate for your user(s)? 
+
+A DSL would be appropriate for my users because what they lack is the translation from the name of a chord to visualizing 1) the notes that comprise the chord and 2) the intervals between the different notes. Chords are typically described with a the root, quality, and inversion -- which is a language itself. My DSL would parse that input and translate it into a language that a program such as LilyPad would understand. 
+
+The domain specificity of this language is important; this language does not have to perform the function of a general programming language, but what it should do is provide just enough functionality for users to input the name of a chord and as output, see the chord dictated on a musical staff. If the language was not domain specific, the learning curve might be unnecessarily steep for the function/goal of the language, and might involve a lot of unnecessary elements that are unrelated to the user’s goals. To eliminate unnecessary functionality, I want to create a 
+
+# How does it address the need?
+
+Again, my users are people who can sight read sheet music but may not know much music theory. My DSL addresses the need because it translates from the name of a musical chord to a visual display of the chord mapped onto sheet music. The chord is in the form of sheet music, which is a better representation of a chord than the "name" of the chord, which is what traditional chord sheets offer. This output that my DSL offers is a more accessible and readable version of chord sheets that the user would otherwise have to struggle to read and understand before he/she can play the chord on the piano. My DSL will hopefully help people find chord sheets more accessible! 
```

Comment:
This is a little surprising to me. I always assumed that to be able to sight read, you'd have to be well-versed in music in general, so I assumed that people who can sight read must understand music theory. But it makes total sense since those skills do not necessarily depend on one another. I didn't know this niche existed.